lib/ex_uid2/api.ex

```diff
@@ -19,8 +19,9 @@ defmodule ExUid2.Api do
   defp post(payload, path) do
     base_url = Application.fetch_env!(:ex_uid2, :base_url)
     api_key = Application.fetch_env!(:ex_uid2, :api_key)
+    req_options = Application.get_env(:ex_uid2, :req_opts, [auth: {:bearer, api_key}])
```

Comment:
This introduces the possibility of a user misconfiguring the application, and not passing a bearer token in requests, even when they are forced to specify an api key in the configuration.

Since the line above insists that the api key is passed separately, you may want to always add the auth stuff on top of whatever config is passed in.

Reply:
Done.